mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -604,8 +604,12 @@ def set_registered_model_tag(self, name, tag):
         _validate_registered_model_tag(tag.key, tag.value)
         with self.ManagedSessionMaker() as session:
             # check if registered model exists
-            self._get_registered_model(session, name)
+            updated_time = get_current_time_millis()
+            sql_model_version = self._get_sql_model_version(session, name=name)
+            sql_model_version.last_updated_time = updated_time
```

Comment:
Should changing a RegisteredModel tag update the ModelVersion updated_time? That's not very intuitive to me.